### PR TITLE
Changes in Stack, Row and List

### PR DESCRIPTION
**Pull request by raquelarrojo**
- Stack, Row and List types moved to an external file.
- Change in Stack, Row and List stories.

### CODE REVIEW

File: lib/src/bleed/Bleed.tsx [hunk 1]
```diff
@@ -54,8 +54,8 @@ function getSpacingValue(spacingName) {
   }
 }
 
-const StyledBleed = styled.div<BleedProps>`
-  ${({ space, horizontal, vertical, top, right, bottom, left }) => `
+const StyledBleed = styled.div<BleedPropsType>`
+  ${({ space, horizontal, vertical, top, right, bottom, left }: BleedPropsType) => `
```
Comment (marcialfps) [inline lib/src/bleed/Bleed.tsx:58]: It is not necessary to indicate `BleedPropsType` when passing to styled components.

File: lib/src/row/Row.tsx [hunk 2]
```diff
@@ -19,9 +18,9 @@ export default function Row({ gutter = "none", align, justify, wrap = true, reve
 }
 const StyledRow = styled.div`
   display: flex;
-  flex-direction: ${({ reverse }: RowProps) => (reverse ? "row-reverse" : "row")};
-  flex-wrap: ${({ wrap }: RowProps) => (wrap ? "wrap" : "nowrap")};
-  align-items: ${({ align }: RowProps) => {
+  flex-direction: ${({ reverse }: RowPropsType) => (reverse ? "row-reverse" : "row")};
+  flex-wrap: ${({ wrap }: RowPropsType) => (wrap ? "wrap" : "nowrap")};
+  align-items: ${({ align }: RowPropsType) => {
```
Comment (marcialfps) [inline lib/src/row/Row.tsx:23]: It is not necessary to indicate `RowPropsType` in all cases if you indicate it to styled components.

File: lib/src/stack/Stack.tsx [hunk 3]
```diff
@@ -32,7 +25,7 @@ const Divider = styled.div`
 const StyledStack = styled.div`
   display: flex;
   flex-direction: column;
-  align-items: ${({ align }: StackProps) => {
+  align-items: ${({ align }: StackPropsType) => {
```
Comment (marcialfps) [inline lib/src/stack/Stack.tsx:28]: Same.

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: lib/src/bleed/Bleed.tsx (hunk 1 above)
Comment (raquelarrojo) [reply lib/src/bleed/Bleed.tsx:58]: Getting this error if I only indicate `BleedPropsType` in the styled component:

image

The same for `Row` and `Stack`.

Comment (marcialfps) [reply lib/src/bleed/Bleed.tsx:58]: I don't have that error. 
image

We are using the same approach in the radio group and is working fine...

Comment (raquelarrojo) [reply lib/src/bleed/Bleed.tsx]: Done!

File: lib/src/stack/Stack.tsx (hunk 3 above)
Comment (raquelarrojo) [reply lib/src/stack/Stack.tsx]: Done!

File: lib/src/row/Row.tsx (hunk 2 above)
Comment (raquelarrojo) [reply lib/src/row/Row.tsx]: Done!

**Review by marcialfps — APPROVED** [review]
💯

**Pull request merged**